src/index.ts

```diff
@@ -136,7 +136,11 @@ export { default as common_button_fr } from './components/translations/common-bu
 export { default as directory_items_input_en } from './components/translations/directory-items-input-en';
 export { default as directory_items_input_fr } from './components/translations/directory-items-input-fr';
 
-export { TagRenderer } from './components/ElementSearchDialog';
+export {
+    TagRenderer,
+    ElementSearchInput,
+    useElementSearch,
+} from './components/ElementSearchDialog';
```

Comment:
Should we export `ElementSearchDialog` as well ?

Reply:
It's already done this way : 
export { default as ElementSearchDialog } from './components/ElementSearchDialog';

I haven't changed this behaviour here